packages/dd-trace/src/appsec/index.js

```diff
@@ -35,9 +36,10 @@ function enable (_config) {
 
     Reporter.setRateLimit(_config.appsec.rateLimit)
 
+    bodyParser.subscribe(onRequestBodyParsed)
+    graphqlFinishExecute.subscribe(onGraphqlFinishExecute)
     incomingHttpRequestStart.subscribe(incomingHttpStartTranslator)
     incomingHttpRequestEnd.subscribe(incomingHttpEndTranslator)
-    bodyParser.subscribe(onRequestBodyParsed)
     queryParser.subscribe(onRequestQueryParsed)
```

Comment:
i understand you want to make everything alphabetically ordered, but the listerners functions declarations are not sorted like that, so either make them sorted too, or keep the original sorting by time of implementation. (imo by time of implementation is better)